app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+    has_many :posts, dependent: :destroy
```

Comment:
- [ ] As mentioned by the previous reviewer, the `User` model `has_many` posts models through the `author_id`. Kindly state it here so that the associations are properly defined. 👍 😎